src/transactions/feed/SwapFeedItem.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import SwapIcon from 'src/icons/SwapIcon'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { useTokenInfo } from 'src/tokens/hooks'
+import { TokenExchange } from 'src/transactions/types'
+
+interface Props {
+  exchange: TokenExchange
+}
+
+function SwapFeedItem({ exchange }: Props) {
+  const { t } = useTranslation()
+  const incomingTokenInfo = useTokenInfo(exchange.inAmount.tokenAddress)
+  const outgoingTokenInfo = useTokenInfo(exchange.outAmount.tokenAddress)
+
+  const openTransferDetails = () => {
+    navigate(Screens.TransactionDetailsScreen, { transaction: exchange })
+    ValoraAnalytics.track(HomeEvents.transaction_feed_item_select)
+  }
+
+  return (
+    <Touchable disabled={false} onPress={openTransferDetails}>
+      <View style={styles.container}>
+        <View style={styles.iconContainer}>
+          <SwapIcon />
+        </View>
+        <View style={styles.contentContainer}>
+          <Text style={styles.title} testID={'SwapFeedItem/title'}>
+            {t('swapScreen.title')}
+          </Text>
+          <Text style={styles.subtitle} testID={'SwapFeedItem/subtitle'}>
+            {outgoingTokenInfo?.symbol} {'to'} {incomingTokenInfo?.symbol}
+          </Text>
+        </View>
+        <View style={styles.contentContainer}>
+          <TokenDisplay
+            amount={exchange.inAmount.value}
+            tokenAddress={exchange.inAmount.tokenAddress}
+            showLocalAmount={false}
+            showSymbol={true}
+            showExplicitPositiveSign={true}
+            hideSign={false}
+            style={[styles.amount, { color: colors.greenUI }]}
```

Comment:
i think this can use `tokenAmount` for the style, and an extra style to create padding on the bottom + add the green color

Reply:
If I change it according to your recommendation, it is not like the figma design.

Current design.
<img width="458" alt="Screen Shot 2022-08-12 at 1 37 16 PM" src="https://user-images.githubusercontent.com/47931506/184720293-33111256-3ce5-43c8-9b42-45a86116ebe5.png">

Your recommendation.
<img width="454" alt="Screen Shot 2022-08-15 at 2 17 30 PM" src="https://user-images.githubusercontent.com/47931506/184720324-6ca29ba6-52f7-4045-87fa-937484fac237.png">

Figma design (https://www.figma.com/file/jRtYI8GiSvPAaClBnRLLpu/In-App-Swap?node-id=1471%3A13632)
<img width="729" alt="Screen Shot 2022-08-15 at 2 15 58 PM" src="https://user-images.githubusercontent.com/47931506/184720365-ea84b96b-06c5-43ee-b36d-ad485b55d48e.png">